openproblems/tasks/chromatin_potential/metrics/correlation.py

```diff
@@ -0,0 +1,25 @@
+from scipy.stats import pearsonr
+from scipy.stats import spearmanr
+from scipy.sparse import issparse
+import numpy as np
+from ....tools.decorators import metric
+
+
+@metric(metric_name="Correlation between RNA and ATAC", maximize=True)
+def correlation(adata, method="pearson"):
```

Comment:
the `method` kwarg will never be used. Better to have a `_correlation` function that defines both and then a `metric` called `pearson_correlation` and a `metric` called `spearman_correlation`.